src/OrchardCore.Modules/OrchardCore.Facebook/ScriptsMiddleware.cs

```diff
@@ -0,0 +1,69 @@
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using OrchardCore.Entities;
+using OrchardCore.Facebook.Settings;
+using OrchardCore.Settings;
+
+namespace OrchardCore.Facebook
+{
+    public class ScriptsMiddleware
+    {
+        private readonly RequestDelegate _next;
+        private readonly ISiteService _siteService;
+
+        public ScriptsMiddleware(RequestDelegate next, ISiteService siteService)
+        {
+            _next = next;
+            _siteService = siteService;
+        }
+
+        public async Task Invoke(HttpContext httpContext)
+        {
+            string script = null;
+            if (httpContext.Request.Path.StartsWithSegments("/OrchardCore.Facebook/sdk"))
+            {
+                var site = (await _siteService.GetSiteSettingsAsync());
+                var settings = site.As<FacebookSettings>();
+                if (httpContext.Request.Path.Value.EndsWith("fbsdk.js"))
+                {
+                    var locale = string.IsNullOrWhiteSpace(site.Culture) ? "en_US" : site.Culture.Replace("-", "_");
```

Comment:
This settings doesn't exist anymore. You need to use `ILocalizationService` and it will give you the `DefaultCulture` to use.
But I think you should use Globalinfo.CurrentUICulture instead.